lineman/app/models/discussion_model.coffee

```diff
@@ -9,6 +9,7 @@ angular.module('loomioApp').factory 'DiscussionModel', (BaseModel, AppConfig) ->
     afterConstruction: ->
       if @isNew()
         @private = @privateDefaultValue()
+      @volume = @discussionReaderVolume or @recordStore.memberships.find({userId: AppConfig.currentUserId, groupId: @groupId })[0].volume
```

Comment:
At first look I think this should be a function.. but now I understand you're doing it so we can query on this. Seems like the right thing to do, but if membership volume changes, what do we do?